packages/types/custom/core.ts

```diff
@@ -32,7 +32,7 @@ export interface PluginOptions {
   layoutTag?: string // TODO: Might it be useful to move declaration of NineLayoutTag here?
 }
 
-export type RenderType = 'iconMenu' | 'independent'
+export type RenderType = 'iconMenu' | 'independent' | 'footer'
```

Comment:
Please also add a changelog entry for the `@polar/lib-custom-types` packages

Reply:
description added